### PR TITLE
print parser error at the end of example parser client

### PR DESCRIPTION
**Pull request by nils-ohlmeier**

### CODE REVIEW

File: examples/file_parser.rs [hunk 1]
```diff
@@ -27,5 +27,10 @@ fn main() {
         Ok(s) => s,
     };
 
-    assert!(webrtc_sdp::parse_sdp(&s, true).is_ok());
+    let res = webrtc_sdp::parse_sdp(&s, true);
+    if res.is_ok() {
+        println!("successfully parsed SDP.");
+    } else {
+        println!("FAILURE: {}", res.err().unwrap());
```
Comment (na-g) [inline examples/file_parser.rs]: `println!(...)` will output to `standard out`, we probably want that message to end up on `standard error`. Elsewhere we are using `panic!` so we can return the Error result if we want.

```rust
fn main() -> Result<(), webrtc_sdp::SdpParserError> {
...
webrtc_sdp::parse_sdp(&s, true).or_else(|e| { eprintln!("Failed to parse"); Err(e) });
println!("Sucessfully parsed SDP.");
```

Comment (nils-ohlmeier) [reply examples/file_parser.rs]: Thank you. Makes all sense. Addressed.

File: examples/file_parser.rs [hunk 2]
```diff
@@ -8,8 +8,8 @@ extern crate webrtc_sdp;
 fn main() {
     let filename = match env::args().nth(1) {
         None => {
-            println!("Missing file name argument!");
-            return;
+            eprintln!("Missing file name argument!");
+            std::process::exit(1);
```
Comment (na-g) [inline examples/file_parser.rs]: Great, is this being run in CI currently?

Comment (nils-ohlmeier) [reply examples/file_parser.rs]: Do you mean if the file_parser gets invoked/run in CI at all?
The answer would be no.

**Pull request merged**